### PR TITLE
Fix alignment on scrollbar issues for fissures

### LINKED ISSUE #138: Scrollbar visible on fissures
### Scrollbar visible on fissures
---

**Summary (short):**
Scrollbar visible on fissures when using certain display modes with small viewports.

---
**Description:**
Happens on my 1080p monitors on different machines, but not on my Mac's 13" built-in display.

---
**If issue, Reproduction:**
Reduce viewport size until they appear.

---
**Mockups, screenshots, evidence:**
capture

---

(Check one)
- [x] Issue
- [ ] Suggestion

### PR DESCRIPTION
**Pull request by dsinn**
Closes #138.

### Fix alignment on scrollbar issues for fissures
---
**Summary (short):**
See issue.

---
**Description:**
See issue.

---
**Fixes issue (include link):**
#138 

---
**Mockups, screenshots, evidence:**
See issue.

---

(Check one)
- [x] Issue fix
- [ ] Suggestion fulfillment

### CODE REVIEW

File: assets/js/updaters.js [hunk 1]
```diff
@@ -734,15 +734,16 @@ function updateFissure() {
...
-        // fissure timer
-        fissureRow += `<span id="fissuretimer${fissure.id}" class="label timer pull-right" data-starttime="${moment(fissure.activation).unix()}" ` +
-                      `data-endtime="${moment(fissure.expiry).unix()}"></span>`;
+        let fissureRow = `<li class="fissure list-group-item list-group-item-borderless" id="${fissure.id}">`;
 
         // fissure body
         fissureRow += `<span class= "fissure-body">${getImage('fissures', {image: fissure.tierNum, title: `Tier ${fissure.tierNum}`, className: 'fissure-icon'})}`;
         fissureRow += `<b>${fissure.node}</b> | ${fissure.missionType} | ${fissure.tier}</span>`;
 
+        // fissure timer
```
Comment (TobiTenno) [inline assets/js/updaters.js:743]: The timer should remain at the beginning so it doesn't get wrapped down at lower resolutions.

**Review by TobiTenno — CHANGES_REQUESTED** [review]

File: assets/js/updaters.js (hunk 1 above)
Comment (dsinn) [reply assets/js/updaters.js:743]: Could you provide a screenshot of the problem?  It looks at least functional to me down to 200px width.

Comment (TobiTenno) [reply assets/js/updaters.js:743]: it's a past problem that i'd prefer avoid happening again. It's hard to explain, and i'll try to see if i can repro again, but i'd prefer we didn't change the order in which they're added if possible.

Comment (TobiTenno) [reply assets/js/updaters.js:743]: looks like it's working fine now... cool. i'll approve and merge.

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**